src/models/target/index.ts

```diff
@@ -1,7 +1,7 @@
 export interface ITaskModel {
   taskName: string;
```

Comment:
Lets use union types for these instead of nullable types, we want the property to exist but be undefined as opposed to the property not existing at all.
```
examples: string[] | undefined
```